src/forms/TimeInput.jsx

```diff
@@ -19,8 +50,91 @@ class TimeInput extends React.Component {
 	* @description called when the input changes, in turn calls the onChange
 	* 	handler prop, if there is one provided (eg supplied by redux-form or DateTimePicker)
 	*/
-	onChange(e) {
-		this.props.onChange && this.props.onChange(e);
+	onChange(partialState) {
+		if (this.props.onChange) {
+			const stateValues = {
+				...this.state,
+				...partialState
+			};
+
+			const v = `${formatDigits(stateValues.hours) % (this.props.is24Hr ? 24 : 12)}:${formatDigits(stateValues.minutes)}`;
```

Comment:
formatDigits or formatHours? this was something I rushed

Reply:
Good catch, it should be formatHours. I'll get that back in there now...